Set wifi-connect to port 80 and web-ui to port 8080 (#14)
balena-io-projects/balena-dash: README.md

> WiFi Connect periodically tests for a functional internet connection, if nothing is found the device sets itself up as a WiFi access point named `balenaDash` that you can join with a mobile device. **Note: due to the fact the built-in WebUI for balenaDash uses port 80, the default captive portal popup screen will not work.**

Reviewer: The bit about the captive portal not working is redundant now, right?

Author: It is!

Author: Removing!

> WiFi Connect periodically tests for a functional internet connection, if nothing is found the device sets itself up as a WiFi access point named `balenaDash` that you can join with a mobile device. **Note: due to the fact the built-in WebUI for balenaDash uses port 80, the default captive portal popup screen will not work.**

Reviewer: ```suggestion
WiFi Connect periodically tests for a functional internet connection, if nothing is found the device sets itself up as a WiFi access point named `balenaDash` that you can join with a mobile device.

> To use WiFi Connect you need to join the network with a mobile device and ignore the captive portal popup. Ensure that you remain connected to the `balenaDash` network and visit the IP address of the device in a browser on port `80`. For example `http://<ip of balenaDash device>`. This will allow you to access WiFi Connect, perform a site survey and join a different WiFi network.

Reviewer: ```suggestion
To use WiFi Connect you need to join the `balenaDash` network and you should see a captive portal popup. If not, ensure that you remain connected to the `balenaDash` network and visit the IP address of the device in a browser on port `80`. For example `http://<ip of balenaDash device>`. This will allow you to access WiFi Connect, perform a site survey and join a different WiFi network.
```
